packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.tsx

```diff
@@ -0,0 +1,143 @@
+import {
+  buildFacet,
+  buildSearchStatus,
+  Facet,
+  FacetOptions,
+  FacetSortCriterion,
+  FacetState,
+  FacetValue,
+  SearchStatus,
+  SearchStatusState,
+} from '@coveo/headless';
+import {registerFacetToStore} from '../../../utils/store';
+import {FacetContainer} from '../facet-container/facet-container';
+import {getFieldValueCaption} from '../../../utils/field-utils';
+import {FacetValueLabelHighlight} from '../facet-value-label-highlight/facet-value-label-highlight';
+import {FacetValueBox} from '../facet-value-box/facet-value-box';
+
+/**
+ * @internal
+ */
+@Component({
+  tag: 'atomic-segmented-facet',
+  styleUrl: 'atomic-segmented-facet.pcss',
+  shadow: true,
+})
+export class AtomicSegmentedFacet
+  implements InitializableComponent, BaseFacet<Facet, FacetState>
+{
+  @InitializeBindings() public bindings!: Bindings;
+  public searchStatus!: SearchStatus;
+  @State()
+  public searchStatusState!: SearchStatusState;
+  @BindStateToController('facet')
+  @State()
+  public facetState!: FacetState;
+  public facet!: Facet;
+  @Element() private host!: HTMLElement;
+  @State() public error!: Error;
+
+  /**
+   * Specifies a unique identifier for the facet.
+   */
+  @Prop({mutable: true, reflect: true}) public facetId?: string;
+  /**
+   * The field whose values you want to display in the facet.
+   */
+  @Prop({reflect: true}) public field!: string;
+  /**
+   * The non-localized label for the facet.
+   */
+  @Prop({reflect: true}) public label = 'no-label';
+  /**
+   * Whether to exclude the parents of folded results when estimating the result count for each facet value.
+   */
+  @Prop({reflect: true}) public filterFacetCount = true;
+
+  @Prop() public injectionDepth = 1000;
+  /**
+   * The number of values to request for this facet.
+   * Also determines the number of additional values to request each time more values are shown.
+   */
+  @Prop({reflect: true}) public numberOfValues = 6;
+  /**
+   * The sort criterion to apply to the returned facet values.
+   * Possible values are 'score', 'alphanumeric', 'occurrences', and 'automatic'.
+   */
+  @Prop({reflect: true}) public sortCriteria: FacetSortCriterion = 'automatic';
+
+  public initialize() {
+    this.searchStatus = buildSearchStatus(this.bindings.engine);
+    const options: FacetOptions = {
+      facetId: this.facetId,
+      field: this.field,
+      numberOfValues: this.numberOfValues,
+      sortCriteria: this.sortCriteria,
+      facetSearch: {numberOfValues: this.numberOfValues},
+      filterFacetCount: this.filterFacetCount,
+      injectionDepth: this.injectionDepth,
+    };
+    this.facet = buildFacet(this.bindings.engine, {options});
+    this.facetId = this.facet.state.facetId;
+    registerFacetToStore(this.bindings.store, 'facets', {
+      label: this.label,
+      facetId: this.facetId!,
+      element: this.host,
+    });
+  }
+
+  private renderValuesContainer(children: VNode[]) {
+    const classes = `mt-3 ${'box-container'}`;
```

Comment:
I'd remove this line until the styling is added or change to
```suggestion
    const classes = 'mt-3 box-container';
```